engine/apps/api/views/team.py

```diff
@@ -14,7 +13,13 @@
 from common.api_helpers.mixins import PublicPrimaryKeyMixin
 
 
-class TeamViewSet(PublicPrimaryKeyMixin, mixins.ListModelMixin, mixins.UpdateModelMixin, viewsets.GenericViewSet):
+class TeamViewSet(
+    PublicPrimaryKeyMixin,
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
```

Comment:
`mixins.RetrieveModelMixin` is added here to add a `GET /team/{teamId}` route. This is needed by the Grafana Incident team for their Add Responders work. See tests [here](https://github.com/grafana/oncall/pull/3241/files#diff-4dd431eecd82d7c2032de7aba5c35a3b6b099ca35ee1abb2cdcdc6f4acac6e3eR31-R47)